tests/test_config_consistency.py

```diff
@@ -293,6 +311,7 @@ def _get_span_by_tags(spans, tags):
 
             if k not in meta:
                 logger.debug(f"Span {span['span_id']} does not have tag {k}")
+                break
```

Comment:
This change is intented ?

Reply:
Yes. We only want to return the span if all of the tags exist and have the right value.